src/Microsoft.SqlTools.ServiceLayer/Migration/Contracts/StartLoginMigration.cs

```diff
@@ -0,0 +1,44 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.SqlServer.DataCollection.Common.Contracts.OperationsInfrastructure;
+using Microsoft.SqlTools.Hosting.Protocol.Contracts;
+using System.Collections.Generic;
+
+namespace Microsoft.SqlTools.ServiceLayer.Migration.Contracts
+{
+    public class StartLoginMigrationParams
+    {
+        /// <summary>
+        /// Connection string to connect to source 
+        /// </summary>
+        public string SourceConnectionString { get; set; }
+
+        /// <summary>
+        /// Connection string to connect to target
+        /// </summary>
+        public string TargetConnectionString { get; set; }
+
+        /// <summary>
+        /// List of logins to migrate
+        /// </summary>
+        public List<string> LoginList { get; set; }
```

Comment:
Would you also need something like CSVFilePath of logins in case user wants to send a large batch of logins for migrations? That is what the NuGet uses for console app/PowerShell.

Reply:
So the UI doesn't give CX the option of passing in a CSVFilePath. Right now the workflow lists all the logins and the CX needs to select which logins to migrate. Do we want to add this functionality as well?

Reply:
Oh you're right sorry, forgot that the UI will have table list of logins that are selectable. Please ignore comment!